src/main/java/me/mrCookieSlime/Slimefun/Lists/SlimefunItems.java

```diff
@@ -404,6 +404,7 @@ private SlimefunItems() {}
     public static final ItemStack SILVER_INGOT = new SlimefunItemStack("SILVER_INGOT", Material.IRON_INGOT, "&bSilver Ingot");
     public static final ItemStack ALUMINUM_INGOT = new SlimefunItemStack("ALUMINUM_INGOT", Material.IRON_INGOT, "&bAluminum Ingot");
     public static final ItemStack LEAD_INGOT = new SlimefunItemStack("LEAD_INGOT", Material.IRON_INGOT, "&bLead Ingot");
+    public static final SlimefunItemStack LEAD_PLATE = new SlimefunItemStack("LEAD_PLATE", Material.PAPER, "&b&lLead Plate");
```

Comment:
This should probably be moved to one of the other plates rather than sticking it in the middle of the "Ingots" section.